test_utils/setupTests.js

```diff
@@ -18,6 +18,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import '@testing-library/jest-dom/extend-expect' // Add extra expect functions to be used in tests
```

Comment:
Nice moving this here so we don't need to import it everywhere, I see there's still an import in `FeatureToggle.test.tsx` should we remove that one as well?

Reply:
Well spotted, thanks!